### PR TITLE
Changes affected the delivery statuses

### PR DESCRIPTION
**Pull request by Gelnas**

### CODE REVIEW

File: src/main/java/com/training/core/model/DeliveryStatus.java [hunk 1]
```diff
@@ -1,10 +1,43 @@
 package com.training.core.model;
 
 public enum DeliveryStatus {
+    /**
+     * Delivery record is open.
```
Comment (ValeriyEmelyanov) [inline src/main/java/com/training/core/model/DeliveryStatus.java:5]: The order for delivery is registered.

File: src/main/java/com/training/core/model/DeliveryStatus.java [hunk 2]
```diff
@@ -1,10 +1,43 @@
...
+     */
     REGISTERED,
+
+    /**
+     * Delivery is paid.
```
Comment (ValeriyEmelyanov) [inline src/main/java/com/training/core/model/DeliveryStatus.java:10]: The ...

File: src/main/java/com/training/core/model/DeliveryStatus.java [hunk 3]
```diff
@@ -1,10 +1,43 @@
...
+     */
     PAID,
-    SENT,
-    COURIER_DELIVERY,
+
+    /**
+     * Cargo in transfer.
```
Comment (ValeriyEmelyanov) [inline src/main/java/com/training/core/model/DeliveryStatus.java:15]: The cargo is in transfer.

File: src/main/java/com/training/core/model/DeliveryStatus.java [hunk 4]
```diff
@@ -1,10 +1,43 @@
...
+     */
+    IN_TRANSFER,
+
+    /**
+     * Cargo is in the destination city. Forwarded to courier delivery.
```
Comment (ValeriyEmelyanov) [inline src/main/java/com/training/core/model/DeliveryStatus.java:20]: The ...

File: src/main/java/com/training/core/model/DeliveryStatus.java [hunk 5]
```diff
@@ -1,10 +1,43 @@
...
+    ON_RETURN,
+
+    /**
+     *  The cargo was delivered. Shipping is closed.
+     */
     DELIVERED,
-    RETURNED
+
+    /**
+     * The cargo was returned to the sender.
```
Comment (ValeriyEmelyanov) [inline src/main/java/com/training/core/model/DeliveryStatus.java:35]: ... Shipping is closed.

File: src/main/java/com/training/core/model/DeliveryStatus.java [hunk 6]
```diff
@@ -1,10 +1,43 @@
...
+     */
+    RETURNED,
+
+    /**
+     * The cargo was lost
```
Comment (ValeriyEmelyanov) [inline src/main/java/com/training/core/model/DeliveryStatus.java:40]: ... . Shipping is closed.

**Review by ValeriyEmelyanov — APPROVED** [review]

**Review by vibriss — APPROVED** [review]

**Pull request merged**